web/meowmentum/src/app/(main)/layout.tsx

```diff
@@ -7,6 +7,10 @@ export default function MainLayout({
   children: React.ReactNode;
 }) {
   return (
+    // <div className="flex flex-row h-[100vh]">
```

Comment:
what is this?

Reply:
Forgot to delete comments for testing purposes